spec/requests/content_item_requests/downstream_requests_spec.rb

```diff
@@ -218,15 +218,16 @@
     end
 
     it "doesn't send draft dependencies to the live content store" do
-      expect(PublishingAPI.service(:live_content_store)).to receive(:put_content_item)
-        .with(a_hash_including(base_path: '/a'))
+      allow(PublishingAPI.service(:live_content_store)).to receive(:put_content_item)
+      allow(PublishingAPI.service(:draft_content_store)).to receive(:put_content_item)
```

Comment:
Yes, an `expect` was changed to an `allow` as the expectation didn't relate to what the test said it would test.